tests/transform/conftest.py

```diff
@@ -10,12 +10,25 @@
 
 
 @pytest.fixture(scope="function")
-def cratedb(cratedb_service):
+def cratedb_custom_service():
+    """
+    Provide a CrateDB service instance to the test suite.
+    """
+    from cratedb_toolkit.testing.testcontainers.cratedb import CrateDBTestAdapter
+
+    db = CrateDBTestAdapter(crate_version="5.8.3")
+    db.start()
+    yield db
+    db.stop()
```

Comment:
Needed to downgrade to CrateDB 5.8.3 in order to satisfy CI.

- https://github.com/crate/crate/issues/16734